src/SDKs/SecurityCenter/Management.SecurityCenter/Microsoft.Azure.Management.SecurityCenter.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.SecurityCenter</PackageId>
+    <Description>Provides developers with libraries for the updated Azure Security Center platform under Azure Resource manager to view and manage security posture in and outside Azure.</Description>
+    <Version>0.1.0</Version>
```

Comment:
Looks like the SDK is still in preview version. The Nuget package must also be in preview. Preview versioning must start from
`0.9.0-preview`

Reply:
changed to `0.9.0-preview`